### PR TITLE
Bulk tasks

### PR DESCRIPTION
**Pull request by rtibbles**
### Summary
Adds bulk endpoints for:
* Remote Import
* Disk Import
* Disk Export
* Deletion
Of channels and content together.

### Reviewer guidance
POST:
```
{
    "channel_ids": [....],
}
```
Is the main form, with additional metadata depending on the endpoint.

I excluded node_ids and excluded node_ids arguments from the bulk import/export endpoint, as I assumed it was unneeded. If this would be helpful, then we could change it to accept just an array of objects that the non-bulk equivalents would accept.

To support: https://www.notion.so/learningequality/Multi-channel-import-and-export-9cd8a0216be74cda84401ac496fb2968

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/tasks/api.py [hunk 1]
```diff
@@ -71,6 +71,56 @@ def destroy(self, request, pk=None):
...
+        except KeyError:
+            raise serializers.ValidationError("The channel_ids field is required.")
+
+        baseurl = request.data.get(
+            "baseurl", conf.OPTIONS["Urls"]["CENTRAL_CONTENT_BASE_URL"]
+        )
+
+        channel_job_metadata = {
+            "type": "REMOTECHANNELIMPORT",
+            "started_by": request.user.pk,
```
Comment (jonboiser) [inline kolibri/core/tasks/api.py]: add the channel ID as well as the number of resources and total size

File: kolibri/core/tasks/api.py [hunk 2]
```diff
@@ -138,6 +188,64 @@ def startremotecontentimport(self, request):
...
+        try:
+            drive = get_mounted_drive_by_id(drive_id)
+        except KeyError:
+            raise serializers.ValidationError(
+                "That drive_id was not found in the list of drives."
+            )
+
+        channel_job_metadata = {
+            "type": "DISKCHANNELIMPORT",
+            "started_by": request.user.pk,
```
Comment (jonboiser) [inline kolibri/core/tasks/api.py]: add channel id plus file size, no. resouces

File: kolibri/core/tasks/api.py [hunk 3]
```diff
@@ -223,6 +331,31 @@ def startdiskcontentimport(self, request):
...
         return Response(resp)
 
+    @list_route(methods=["post"])
+    def startbulkdelete(self, request):
+        try:
+            channel_ids = request.data["channel_ids"]
+        except KeyError:
+            raise serializers.ValidationError("The channel_ids field is required.")
+
+        job_metadata = {"type": "DELETECHANNEL", "started_by": request.user.pk}
```
Comment (jonboiser) [inline kolibri/core/tasks/api.py]: like for import tasks, add extra metadata here

File: kolibri/core/assets/src/api-resources/task.js [hunk 4]
```diff
@@ -89,6 +127,12 @@ export default new Resource({
     });
   },
 
+  deleteBulkChannels(channelIds) {
```
Comment (jonboiser) [inline kolibri/core/assets/src/api-resources/task.js]: Have this one take an object argument `{ channel_ids }` so its consistent with other functions

**Comment by rtibbles** [comment]
Have updated to provide additional metadata where possible, and allow adding the metadata from the frontend otherwise.

**Review by jonboiser — APPROVED** [review]

**Pull request merged**